DAQ_System/Source/main.cpp

```diff
@@ -27,205 +26,236 @@
  */
 #include "mbed.h"
 #include <stdio.h>
+#include <cstdio>
 #include <errno.h>
 
-// Block devices
-//#include "SPIFBlockDevice.h"
-//#include "DataFlashBlockDevice.h"
 #include "SDBlockDevice.h"
-//#include "HeapBlockDevice.h"
-
-// File systems
-
-//#include "LittleFileSystem.h"
 #include "FATFileSystem.h"
 
-// Physical block device, can be any device that supports the BlockDevice API
-SDBlockDevice   blockDevice(PA_7, PA_6, PA_5, PB_6);  // mosi, miso, sck, cs
-
-// File system declaration
-FATFileSystem   fileSystem("fs");
+void mount(FATFileSystem*, SDBlockDevice*);
+void unmount(FATFileSystem*);
+FILE* file_open(const char*);
+void file_close(FILE*);
+void file_write(FILE*, const char*);
```

Comment:
I looked back at the C++ style guide because I thought they had used snake case for variables and functions, but for functions they actually use all capitals, do you have a preference for camelCase or should we capitalize the first word like the style guide says?

Reply:
> Camel Case (including the first letter)

I guess the word I was looking for is PascalCase. Go ahead capatilize the first letter of every word, without underscores ([here are some examples](https://google.github.io/styleguide/cppguide.html#Function_Names))